packages/bun-types/bun.d.ts

```diff
@@ -2354,12 +2369,12 @@ declare module "bun" {
      * ```js
      * export default {
      *  async fetch(request, server) {
-     *    return new Response(server.requestIp(request));
+     *    return new Response(server.requestIP(request));
```

Comment:
![](https://media.tenor.com/5U3zBvjgk2gAAAAC/thank-you.gif)

Reply:
Why? In the js world we use camelCase so why was this accepted?

Reply:
If you look at the official nodejs documentation https://nodejs.org/dist/latest-v20.x/docs/api/http.html you will find that all acronyms like `http` are not capitalised besides the first letter when used in `camelCase`. I think we should align ourself with that and not start writing acronyms in caps. Besides that, currently bun has no capitalised acronyms either so this would be our (mis)first step towards that path.

Reply:
If the goal of bun is to be better than nodejs, then why should they not take every opportunity to drop bad conventions of the past. They can set new conventions and do things correctly like not write acronyms as if they were words.

Reply:
> Why? In the js world we use `camelCase` so why was this accepted?

it was decided from: https://twitter.com/bunjavascript/status/1707690336494186919

Reply:
The thing is this is a JavaScript convention not a nodeJS convention. As bun resides in the JS/TS world I think we should stick to that.